attendance/test/api_test.dart

```diff
@@ -168,4 +168,54 @@ void main() {
       }
     });
   });
+
+  group('test getting attendance', () {
+    final Future<String> t = token();
+    const String _attendanceKey = '-LXDFDibiC8EyKRg96Zv';
```

Comment:
@ammaratef45 Isn't it better if we created a generator class that bind these values without hardcoding them?

Reply:
@Bahaaib I'll leave a puzzle for it